nuxeo-features/nuxeo-automation/nuxeo-automation-features/src/main/java/org/nuxeo/ecm/automation/core/operations/collections/GetCollectionsOperation.java

```diff
@@ -1,5 +1,5 @@
 /*
- * (C) Copyright 2014-2019 Nuxeo SA (http://nuxeo.com/) and others.
+ * (C) Copyright 2014 Nuxeo SA (http://nuxeo.com/) and others.
```

Comment:
Should be part of cleanup commit.
Or do we want 2019 only? Or just leave it as is?

Reply:
I ended up not touching the file, so I removed the header update :o

Reply:
So finally you kept the update?

Reply:
I took it of the cleanup so it doesn't show at all anymore